pkg/adminendpoint/adminendpoint.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
add another line above: `Copyright (c) 2019 The Jaeger Authors.`

Reply:
Shouldn't it be only "The Jaeger Authors" for new files, without "Uber Technoligies, Inc"?

Reply:
Yes, but a portion of this file is a copy